### PR TITLE
[plugin.video.ign_com] 2.3.7

### PR DESCRIPTION
**Pull request by skipmodea1**
### Description
2.3.7 (2018-09-08)
- website switched to https

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by skipmodea1** [comment]
hmm, i don't get what's wrong with the addon.xml :/

**Comment by Rechi** [comment]
`disclaimer` tag misses the `lang` attribute

File: plugin.video.ign_com/addon.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <addon
 	id="plugin.video.ign_com"
 	name="IGN.com"
-	version="2.3.6"
+	version="2.3.7"
 	provider-name="Skipmode A1">
     <requires>
         <import addon="xbmc.python"                     version="2.14.0"/>
```
Comment (Rechi) [inline plugin.video.ign_com/addon.xml]: > `WARN: For helix it is advised to set xbmc.python version to 2.19.0`

File: plugin.video.ign_com/addon.xml [hunk 2]
```diff
@@ -13,21 +13,20 @@
         <provides>video</provides>
     </extension>
     <extension point="xbmc.addon.metadata">
-        <summary lang="en">Game and Movie trailers/reviews/news</summary>
-        <description lang="en">Game and Movie trailers/reviews/news</description>
+        <summary lang="en_GB">Game and Movie trailers/reviews/news</summary>
+            <description lang="en_GB">Game and Movie trailers/reviews/news</description>
+            <disclaimer lang="en_GB">For bugs, requests or general questions visit the IGN_COM thread on the Kodi forum.</disclaimer>
+        <language>en_GB</language>
```
Comment (Rechi) [inline plugin.video.ign_com/addon.xml]: I don't think this is correct.

Comment (skipmodea1) [reply plugin.video.ign_com/addon.xml]: what's wrong with it?

Comment (Rechi) [reply plugin.video.ign_com/addon.xml]: I think `en` is correct, but haven't checked Kodi core code.

Comment (skipmodea1) [reply plugin.video.ign_com/addon.xml]: ah ok, i'll make another pr, Thanks

Comment (Rechi) [reply plugin.video.ign_com/addon.xml]: Can't you update the existing PR?

**Pull request closed without merging**